src/test/kotlin/org/rust/ide/annotator/fixes/ConvertToSizedTypeFixTest.kt

```diff
@@ -51,6 +51,7 @@ class ConvertToSizedTypeFixTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
             checkWarn = true,
             checkInfo = false,
             checkWeakWarn = false,
+            previewExpected = true,
```

Comment:
Why?

Reply:
I refactored `ConvertToSizedTypeFixTest` so now default value parameter from super method will be used